### PR TITLE
feat(update) aims to improve & fix Menyoo

### PR DESCRIPTION
**Pull request by Clone69420**

### CODE REVIEW

File: ScriptHookVDotNet.LICENSE.txt [hunk 1]
```diff
@@ -1,16 +0,0 @@
...
-authors be held liable for any damages arising from the use of this software.
-Permission  is granted  to anyone  to use  this software  for  any  purpose,  including  commercial
-applications, and to alter it and redistribute it freely, subject to the following restrictions:
-
-  1. The origin of this software must not be misrepresented; you must not claim that you  wrote the
-     original  software. If you use this  software  in a product, an  acknowledgment in the product
-     documentation would be appreciated but is not required.
-  2. Altered source versions must  be plainly  marked as such, and  must not be  misrepresented  as
-     being the original software.
-  3. This notice may not be removed or altered from any source distribution.
```
Comment (tupoy-ya) [inline ScriptHookVDotNet.LICENSE.txt]: This file is probably for some enum or other stuff which i'm too lazy to find.

File: README.md [hunk 2]
```diff
@@ -3,38 +3,55 @@
...
+  ```bash
+  git clone https://github.com/tupoy-ya/MenyooMP.git
+  ```
 
-## Build
-Visual Studio 2017 required
-- I used v15.9.7
+-  Go into the directory you just cloned:
+  ```bash
+  cd MenyooMP
```
Comment (tupoy-ya) [inline README.md]: `cd MenyooMP/Solution`

**Review by tupoy-ya — CHANGES_REQUESTED** [review]

**Comment by Clone69420** [comment]
Is .stignore even needed? Bec you can just put it in .gitignore ig.

**Comment by tupoy-ya** [comment]
> Is .stignore even needed? Bec you can just put it in .gitignore ig.

.stingore is for [syncthing](https://syncthing.net) which i used to sync my windows VM with my host system.

It's not really needed anymore because i use msvc on linux now.

File: OpenV LICENSE.txt [hunk 3]
```diff
@@ -1,16 +1,16 @@
...
-authors be held liable for any damages arising from the use of this software.
-Permission  is granted  to anyone  to use  this software  for  any  purpose,  including  commercial
-applications, and to alter it and redistribute it freely, subject to the following restrictions:
-
-  1. The origin of this software must not be misrepresented; you must not claim that you  wrote the
-     original  software. If you use this  software  in a product, an  acknowledgment in the product
-     documentation would be appreciated but is not required.
-  2. Altered source versions must  be plainly  marked as such, and  must not be  misrepresented  as
-     being the original software.
+https://github.com/crosire/scripthookvdotnet/blob/dev_v2/LICENSE.txt
```
Comment (tupoy-ya) [inline OpenV LICENSE.txt]: Why is this named OpenV LICENSE?

**Comment by Clone69420** [comment]
Idk if u need these weird eGameVersion stuff but should be fine or tested

**Comment by tupoy-ya** [comment]
It's not just MP fixed version of Menyoo it also just improves code readability (not enough yet), eGameVersion stuff is needed if you use an older version of the game (speedrun patch, pirated, just don't update).

About the `_Build` we should put menyooStuff in the root of the repo `MenyooMP/menyooStuff`

File: Solution/source/Memory/GTAmemory.cpp [hunk 4]
```diff
@@ -1010,6 +1010,8 @@ void GeneralGlobalHax::EnableBlockedMpVehiclesInSp(bool uSure)
 	case eGameVersion::VER_1_0_2628_2_NOSTEAM: case eGameVersion::VER_1_0_2628_2_STEAM:
 	case eGameVersion::VER_1_0_2699_0_NOSTEAM: case eGameVersion::VER_1_0_2699_0_STEAM:
 		*GTAmemory::GetGlobalPtr<INT32>(4533757) = uSure ? 1 : 0; break;
+	case eGameVersion::VER_1_0_2699_16_NOSTEAM: case eGameVersion::VER_1_0_2699_16_STEAM:
```
Comment (tupoy-ya) [inline Solution/source/Memory/GTAmemory.cpp]: It's just `VER_1_0_2699_16` without `_STEAM` or `_NOSTEAM` as far as i know.

Here is the enum i have in TupoyeMenu
```cpp
enum eGameVersion : int
{
	VER_1_0_335_2_STEAM,
	VER_1_0_335_2_NOSTEAM,

	VER_1_0_350_1_STEAM,
	VER_1_0_350_2_NOSTEAM,

	VER_1_0_372_2_STEAM,
	VER_1_0_372_2_NOSTEAM,

	VER_1_0_393_2_STEAM,
	VER_1_0_393_2_NOSTEAM,

	VER_1_0_393_4_STEAM,
	VER_1_0_393_4_NOSTEAM,

	VER_1_0_463_1_STEAM,
	VER_1_0_463_1_NOSTEAM,

	VER_1_0_505_2_STEAM,
	VER_1_0_505_2_NOSTEAM,

	VER_1_0_573_1_STEAM,
	VER_1_0_573_1_NOSTEAM,

	VER_1_0_617_1_STEAM,
	VER_1_0_617_1_NOSTEAM,

	VER_1_0_678_1_STEAM,
	VER_1_0_678_1_NOSTEAM,

	VER_1_0_757_2_STEAM,
	VER_1_0_757_2_NOSTEAM,

	VER_1_0_757_4_STEAM,
	VER_1_0_757_4_NOSTEAM,

	VER_1_0_791_2_STEAM,
	VER_1_0_791_2_NOSTEAM,

	VER_1_0_877_1_STEAM,
	VER_1_0_877_1_NOSTEAM,

	VER_1_0_944_2_STEAM,
	VER_1_0_944_2_NOSTEAM,

	VER_1_0_1011_1_STEAM,
	VER_1_0_1011_1_NOSTEAM,

	VER_1_0_1032_1_STEAM,
	VER_1_0_1032_1_NOSTEAM,

	VER_1_0_1103_2_STEAM,
	VER_1_0_1103_2_NOSTEAM,

	VER_1_0_1180_2_STEAM,
	VER_1_0_1180_2_NOSTEAM,

	VER_1_0_1290_1_STEAM,
	VER_1_0_1290_1_NOSTEAM,

	VER_1_0_1365_1_STEAM,
	VER_1_0_1365_1_NOSTEAM,

	VER_1_0_1493_0_STEAM,
	VER_1_0_1493_0_NOSTEAM,

	VER_1_0_1493_1_STEAM,
	VER_1_0_1493_1_NOSTEAM,

	VER_1_0_1604_0_STEAM,
	VER_1_0_1604_0_NOSTEAM,

	VER_1_0_1604_1_STEAM,
	VER_1_0_1604_1_NOSTEAM,
	
	VER_1_0_1737_0_STEAM,
	VER_1_0_1737_0_NOSTEAM,
	
	VER_1_0_1737_6_STEAM,
	VER_1_0_1737_6_NOSTEAM,
	
	VER_1_0_1868_0_STEAM,
	VER_1_0_1868_0_NOSTEAM,
	
	VER_1_0_1868_1_STEAM,
	VER_1_0_1868_1_NOSTEAM,
	VER_1_0_1868_4_EPIC,

	VER_1_0_2060_0_STEAM,
	VER_1_0_2060_0_NOSTEAM,
	
	VER_1_0_2060_1_STEAM,
	VER_1_0_2060_1_NOSTEAM,

	VER_1_0_2189_0_STEAM,
	VER_1_0_2189_0_NOSTEAM,
	
	VER_1_0_2215_0_STEAM,
	VER_1_0_2215_0_NOSTEAM,
	
	VER_1_0_2245_0_STEAM,
	VER_1_0_2245_0_NOSTEAM,

	VER_1_0_2372_0_STEAM,
	VER_1_0_2372_0_NOSTEAM,
	
	VER_1_0_2545_0_STEAM,
	VER_1_0_2545_0_NOSTEAM,

	VER_1_0_2612_0_STEAM,
	VER_1_0_2612_0_NOSTEAM,
	
	VER_1_0_2628_0_STEAM,
	VER_1_0_2628_0_NOSTEAM,

	VER_1_0_2628_2_STEAM,
	VER_1_0_2628_2_NOSTEAM,

	VER_1_0_2699_0_STEAM,
	VER_1_0_2699_0_NOSTEAM,

	VER_1_0_2699_16,
	
	VER_SIZE,
	VER_UNK = -1
};
```

**Comment by tupoy-ya** [comment]
`Solution/source/Menyoo.rc` is required if you load Menyoo in FiveM and probably with normal ScriptHookV

**Review by tupoy-ya — CHANGES_REQUESTED** [review]
Forgot to submit this sorry.

**Pull request closed without merging**